### PR TITLE
Added an option to enable verbose logging

### PR DESCRIPTION
**Pull request by thomasbilk**
I added an option `sftp_enable_logging` that enables verbose logging for
each user.

### CODE REVIEW

**Comment by thomasbilk** [comment]
I checked that again. It does not work as I wanted. You cannot have a separate file for each login/user. I am working on a better setup and will either update this pull request or make a new one tomorrow.

**Comment by thomasbilk** [comment]
So I have updated the configuration. The verbose log cannot be set up individually for every user but logs into a single file for all users. I have also changed the path for the log file so that all SFTP logs are in a single directory.

**Comment by thomasbilk** [comment]
So I tried to fix the conflicts and hope this worked out. Originally I thought I can enable logging per user but that did not work. So now SFTP actions are logged into `/var/log/sftp/verbose.log` and in addition to that authentication is logged into `/var/log/sftp/auth.log`

**Comment by johanmeiring** [comment]
Thanks @thomasbilk, I'll try to take a look at it tonight.

File: tasks/main.yml [hunk 1]
```diff
@@ -39,8 +39,8 @@
           ChrootDirectory %h
           AllowTCPForwarding no
           X11Forwarding no
-          ForceCommand internal-sftp
-          PasswordAuthentication {% if sftp_allow_passwords %}yes{% else %}no{% endif %}
+          ForceCommand internal-sftp {{ sftp_enable_logging | ternary('-l VERBOSE', '') }}
```
Comment (johanmeiring) [inline tasks/main.yml:42]: Cool, didn't know about `ternary`... nice and clean :)

**Comment by johanmeiring** [comment]
Looks good to me, thanks for the contribution.

Some suggestions for future improvements, if you're up for it:
* Allow the locations, ownerships and modes of the two log files to be configurable.
* Disallow `dev` as a creatable directory.

**Comment by markstos** [comment]
This solution is not flexible because the only log level supported is `verbose`. Some more flexible options:

- Support an `sftp_loglevel` string instead of a boolean.
- Support an `sftp_server_args` string, which allow configuring *all* the custom args you desired instead of just the log level.

**Pull request merged**